src/content/components/PriorityGuide/PriorityGuide.module.scss

```diff
@@ -37,17 +37,17 @@
 }
 
 .p1 {
-  background: #45a1ff;
+  background: #45A1FF;
   border-color: transparent;
   color: white;
 }
 .p2 {
-  background: #9fc6ee;
+  background: #9FC6EE;
```

Comment:
I'm confused. These are the same colors, just in uppercase?

Reply:
Sorry I don't know what happened to the modified colors but I'll update the colors again anyway to work in both light and dark mode.